README.md

```diff
@@ -54,6 +54,9 @@ Usage: brew cu [CASK] [options]
     -y, --yes             Update all outdated apps; answer yes to updating packages.
     -q, --quiet           Do not show information about installed apps or current options.
         --no-quarantine   Pass --no-quarantine option to `brew cask install`.
+        --pinned          Print all pinned apps
+        --pin APP         Pin the current app version
```

Comment:
Would be also great if you could write a short description into the README about the pinning, what does it do, etc. Also I would include a warning, that pinning only works if used with `brew cu`, if someone would use directly a `brew cask upgrade` command it won't have any effect.